client/my-sites/checkout/src/lib/translate-payment-method-names.ts

```diff
@@ -1,6 +1,9 @@
+import config from '@automattic/calypso-config';
 import { camelToSnakeCase } from '@automattic/js-utils';
 import type { CheckoutPaymentMethodSlug, WPCOMPaymentMethod } from '@automattic/wpcom-checkout';
 
+const StripeRedirectMigrationIdeal = config.isEnabled( 'stripe-redirect-migration-ideal' );
```

Comment:
Variables that start with a capital are generally reserved for components. It might also help to have the variable name be a little more descriptive of what the variable means (e.g. `isStripeIdealRedirectEnabled`).

Reply:
Oh...I think I remember a very long debugging session with you and Payton about that! Thanks for catching it! I'll get it fixed.

Reply:
✅